Github-Users-List.md

```diff
@@ -0,0 +1,30 @@
+# Mobile Test - Github Users List App
+
+## User Stories
+
+1. As a user on app first launch,
+    - When I reach the home page
+    - Then I should see a list of github users. Each user item on this page should show: name, avatar, the home page URL
+    - And I should see a search box
+2. As a user on the home page,
+    - When I tap on a user item
+    - Then I should see the user's details
+3. As a user on the item details page,
+    - Then I should see user's home page loaded with the home page URL
+4. As a user on the home page,
+    - When I focus on to the search bar
+    - And enter search terms
+    - Then I should see the corresponding news result lists based on the search terms real time
+
+## API
+
+- You can use the example: https://api.github.com/search/users?q=swift
+
+
+## Requirements
+
+1. Swift >=4.2. We are usually interested in how you use Structs, Enums, Extensions, and Protocol Oriented Programming etc.
```

Comment:
This is fine, but I'd just say "We are generally interested in how well you use Swift and it's features such as...<list here>"